manifests/g/GLab/GLab/1.53.0/GLab.GLab.installer.yaml

```
@@ -0,0 +1,28 @@
+# Created with komac v2.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: GLab.GLab
+PackageVersion: 1.53.0
+InstallerLocale: en-US
+InstallerType: inno
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ProductCode: '{679C50AE-F48A-46D1-9493-77F095E9884D}_is1'
+ReleaseDate: 2025-02-14
+AppsAndFeaturesEntries:
+- Publisher: GitLab
+  ProductCode: '{679C50AE-F48A-46D1-9493-77F095E9884D}_is1'
+ElevationRequirement: elevatesSelf
+InstallationMetadata:
+  DefaultInstallLocation: '%ProgramFiles%\glab'
+Installers:
+- Architecture: x64
+  InstallerUrl: https://gitlab.com/gitlab-org/cli/-/releases/v1.53.0/downloads/glab_1.53.0_Windows_x86_64_installer.exe
+  InstallerSha256: 0727626B2498FC2C1303CED716A8DBD7CF25A5D2480074EC00458A942D7D0DD7

```

```suggestion
  InstallerSha256: 00DD4586826C2A9DAEB5950D35319EE7CEF45DB538ED8B3D8F7AF83E14525481
```

(Automated response - build 998.)